### ISSUE TITLE
Should we add new elements for indicating interval durations?

### ISSUE THREAD

**Issue by macintoshpie** [BuildingSync/TestSuite#33]
STD 211, 6.2.2.1.a.6 requires "Number of days in month or period" for Historical Utility Data. Currently, we just require a StartTimestamp and EndTimestamp for monthly reports. I'm guessing we would need to add the element within auc:TimeSeries - below is what we already require plus the new elements
```xml
<auc:TimeSeries ID="Scenario-A-TS1">
  <auc:ReadingType>Total</auc:ReadingType>
  <auc:StartTimestamp>2019-01-01T00:00:00</auc:StartTimestamp>
  <auc:EndTimestamp>2019-02-01T00:00:00</auc:EndTimestamp>
  <!-- add this element -->
  <auc:IntervalDuration>31</auc:IntervalDuration>
  <!-- add this element -->
  <auc:IntervalDurationUnits>Days</auc:IntervalDurationUnits>
  <auc:IntervalFrequency>Month</auc:IntervalFrequency>
  <auc:IntervalReading>4102.51</auc:IntervalReading>
  <auc:ResourceUseID IDref="ResourceUse-Electricity" />
</auc:TimeSeries>
```

Pros of having the number of days:
- now able to do more validation (e.g. annual fuel use = (total number of days / total energy use) * 365)

Cons of having the number of days:
- redundant information which could lead to inconsistent reports

**Comment by macintoshpie** [comment]
We decided to include auc:IntervalDuration and auc:IntervalDurationUnits
For auc:IntervalDurationUnits the enums are the same as those for IntervalFrequency (refactor out into separate type for sharing)

**Comment by macintoshpie** [comment]
Addressed in #37
